hbase-handler/src/test/queries/positive/hbase_ppd_boolean_cols.q

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE hbase_table(row_key string, c1 boolean, c2 boolean)
+STORED BY 'org.apache.hadoop.hive.hbase.HBaseStorageHandler'
+WITH SERDEPROPERTIES (
+"hbase.columns.mapping" = ":key,cf:c1,cf:c2"
+);
+
```

Comment:
even if this is about a compile-time issue, can include a real query too, not just an explain?

Reply:
Fixed in https://github.com/apache/hive/pull/2996/commits/42333553fbac3949e4f0c65e21b6237ca171b971. Please have another look when you find some time.